mnefun/_mnefun.py

```diff
@@ -152,6 +152,14 @@ class Params(object):
         be used for covariance calculation. This is useful e.g. when using
         a high-pass filter and no baselining is desired (but evoked
         covariances should still be calculated from the baseline period).
+    reject_tmin : float | None
+        Reject minimum time to use when epoching. None will use ``tmin``.
+    reject_tmax : float | None
+        Reject maximum time to use when epoching. None will use ``tmax``.
```

Comment:
This is presumably for rejection restricted to an interval WITHIN the epoching interval defined by `tmin` - `tmax`?